ci/prow_setup.md

```diff
@@ -146,3 +108,11 @@
 1. Set the new test jobs as required status checks for the master branch.
 
    ![Branch Checks](branch_checks.png)
+
+## Setting up the issue tracker for a new repo
+
+1. Update [`generateIssueTrackerPeriodicJobs()`](https://github.com/knative/test-infra/blob/51c37921d4a7722855fcbb020db3c3865db1cb8f/ci/prow/issue_tracker_config.go#L48)
+   in [issue_tracker_config.go](./prow/issue_tracker_config.go), adding the call
+   to generate the config for the new repo.
```

Comment:
should we also add `make config` step here?

Reply:
Indeed. Actually there was a bunch of outdated instructions regarding the config, and I updated them too.